src/ResourceManager/Compute/Commands.Compute/Commands.Compute.csproj

```diff
@@ -93,6 +93,18 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Resources.2.18.1-preview\lib\net40\Microsoft.Azure.ResourceManager.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.Data.Edm, Version=5.6.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
You have added new DLLs to the compute library. Please regenerate WIX file to ensure the install is in sync.